trunk/SUAVE/Methods/Weights/Buildups/Common/wing.py

```diff
@@ -85,22 +90,30 @@ def wing(wing,
             weight:                 Wing Mass                           [kg]
     """
 
-#-------------------------------------------------------------------------------
-# Unpack Inputs
-#-------------------------------------------------------------------------------
+    #-------------------------------------------------------------------------------
+    # Unpack Inputs
+    #-------------------------------------------------------------------------------
 
     MTOW                        = config.mass_properties.max_takeoff
     wingspan                    = wing.spans.projected
-    chord                       = wing.chords.mean_aerodynamic,
-    thicknessToChord            = wing.thickness_to_chord, 
-    wingletFraction             = wing.winglet_fraction, 
+    chord                       = wing.chords.mean_aerodynamic
+    thicknessToChord            = wing.thickness_to_chord
     wingArea                    = wing.areas.reference
 
+    try:
+        wingletFraction = wing.winglet_fraction
```

Comment:
Can you add this to the inputs in the doc string?

Reply:
Added